utils/steward-cli/steward_cli_notes.md

```diff
@@ -0,0 +1,296 @@
+  --instant-unstake-delinquency-threshold-ratio 0.70 \
+  --num-delegation-validators 200 \
+  --scoring-unstake-cap-bps 750 \
+  --instant-unstake-cap-bps 1000 \
+  --stake-deposit-unstake-cap-bps 1000 \
+  --compute-score-slot-range 50000 \
+  --instant-unstake-epoch-progress 0.50 \
+  --instant-unstake-inputs-epoch-progress 0.50 \
+  --num-epochs-between-scoring 3 \
+  --minimum-stake-lamports 100000000000 \
+  --minimum-voting-epochs 5
+```
+
+## Create State
+
+```bash
+cargo run -- --program-id sssh4zkKhX8jXTNQz1xDHyGpygzgu2UhcRcUvZihBjP realloc-state --authority-keypair-path ../../credentials/stakenet_test.json --steward-config 35mMfFNiui7hcHy6xHTz11Q6YukbhH9qQgYR5dhWAQQH
+```
+
+## Update Config
+
+```bash
+cargo run -- --program-id sssh4zkKhX8jXTNQz1xDHyGpygzgu2UhcRcUvZihBjP update-config \
+  --authority-keypair-path ../../credentials/stakenet_test.json \
+  --steward-config 35mMfFNiui7hcHy6xHTz11Q6YukbhH9qQgYR5dhWAQQH \
+  --num-epochs-between-scoring 3
+```
+
+## Update Authority
+
+`blacklist` | `admin` | `parameters`
+
+```bash
+cargo run -- --program-id sssh4zkKhX8jXTNQz1xDHyGpygzgu2UhcRcUvZihBjP update-authority blacklist \
+  --authority-keypair-path ../../credentials/stakenet_test.json \
+  --steward-config 35mMfFNiui7hcHy6xHTz11Q6YukbhH9qQgYR5dhWAQQH \
+  --new-authority aaaDerwdMyzNkoX1aSoTi3UtFe2W45vh5wCgQNhsjF8
+```
+
+## Set Staker
+
+```bash
+cargo run -- --program-id sssh4zkKhX8jXTNQz1xDHyGpygzgu2UhcRcUvZihBjP set-staker \
+  --authority-keypair-path ../../credentials/stakenet_test.json \
+  --steward-config 35mMfFNiui7hcHy6xHTz11Q6YukbhH9qQgYR5dhWAQQH
+```
+
+## Revert Staker
+
+```bash
+cargo run -- --program-id sssh4zkKhX8jXTNQz1xDHyGpygzgu2UhcRcUvZihBjP revert-staker \
+  --authority-keypair-path ../../credentials/stakenet_test.json \
+  --steward-config 35mMfFNiui7hcHy6xHTz11Q6YukbhH9qQgYR5dhWAQQH
+```
+
+## Pause
+
+```bash
+cargo run -- --program-id sssh4zkKhX8jXTNQz1xDHyGpygzgu2UhcRcUvZihBjP pause \
+  --authority-keypair-path ../../credentials/stakenet_test.json \
+  --steward-config 35mMfFNiui7hcHy6xHTz11Q6YukbhH9qQgYR5dhWAQQH --print-tx
+```
+
+## Resume
+
+```bash
+cargo run -- --program-id sssh4zkKhX8jXTNQz1xDHyGpygzgu2UhcRcUvZihBjP resume \
+  --authority-keypair-path ../../credentials/stakenet_test.json \
+  --steward-config 35mMfFNiui7hcHy6xHTz11Q6YukbhH9qQgYR5dhWAQQH --print-tx
+```
+
+## Reset State
+
+```bash
+cargo run -- --program-id sssh4zkKhX8jXTNQz1xDHyGpygzgu2UhcRcUvZihBjP reset-state --steward-config 35mMfFNiui7hcHy6xHTz11Q6YukbhH9qQgYR5dhWAQQH  --authority-keypair-path ../../credentials/stakenet_test.json
+```
+
+## View Config
+
+```bash
+cargo run -- --program-id Stewardf95sJbmtcZsyagb2dg4Mo8eVQho8gpECvLx8 view-config --steward-config jitoVjT9jRUyeXHzvCwzPgHj7yWNRhLcUoXtes4wtjv
+```
+
+## View State
+
+```bash
+cargo run -- --json-rpc-url $(solana config get | grep "RPC URL" | awk '{print $3}') view-state --steward-config jitoVjT9jRUyeXHzvCwzPgHj7yWNRhLcUoXtes4wtjv
+```
```

Comment:
would put these view commands at the top since we'll be highlighting these for users

Reply:
Yep!